third_party/move/testing-infra/test-generation/src/lib.rs

```diff
@@ -33,16 +32,13 @@ use move_compiler::{
 };
 use move_core_types::{
     account_address::AccountAddress,
-    effects::{ChangeSet, Op},
     language_storage::TypeTag,
-    resolver::MoveResolver,
     value::MoveValue,
     vm_status::{StatusCode, VMStatus},
 };
 use move_vm_runtime::{module_traversal::*, move_vm::MoveVM};
-use move_vm_test_utils::{DeltaStorage, InMemoryStorage};
+use move_vm_test_utils::InMemoryStorage;
```

Comment:
Ah the old test(module) generation code. It's been broken for years maybe we should get it removed for now so people won't wasting time maintaining it.